core/unit_test/CMakeLists.txt

```diff
@@ -346,7 +347,7 @@ foreach(Tag Threads;Serial;OpenMP;Cuda;HPX;OpenMPTarget;OpenACC;HIP;SYCL)
     # detail tests for the mdspan based View
     set(${Tag}_VIEWSUPPORT)
     if(Kokkos_ENABLE_IMPL_MDSPAN)
-      foreach(Name ReferenceCountedAccessor ReferenceCountedDataHandle)
+      foreach(Name BasicView ReferenceCountedAccessor ReferenceCountedDataHandle)
```

Comment:
@masterleinad is there a cmake-format setting to force writing this like
```cmake
foreach(Name
  BasicView
  ReferenceCountedAccessor
  ReferenceCountedDataHandle
)

Reply:
Sure,
```suggestion
      # cmake-format: off
      foreach(Name
        BasicView
        ReferenceCountedAccessor
        ReferenceCountedDataHandle
      )
      # cmake-format: on
```

Reply:
I meant a configuration option, I know about disabling